lib/process-services/src/lib/task-list/components/task-list/task-list.component.ts

```diff
@@ -61,6 +73,9 @@ export const PRESET_KEY = 'adf-task-list.presets';
     styleUrls: ['./task-list.component.css']
 })
 export class TaskListComponent extends DataTableSchema implements OnChanges, AfterContentInit, PaginatedComponent, OnDestroy, OnInit {
+    @ViewChild(DataTableComponent)
```

Comment:
Why do we need this change?

Reply:
Reverted those changes, as I've found a possibly better solution